sickbeard/__init__.py

```diff
@@ -1126,6 +1127,7 @@ def path_leaf(path):
         TIMEZONE_DISPLAY = check_setting_str(CFG, 'GUI', 'timezone_display', 'network')
         POSTER_SORTBY = check_setting_str(CFG, 'GUI', 'poster_sortby', 'name')
         POSTER_SORTDIR = check_setting_int(CFG, 'GUI', 'poster_sortdir', 1)
+        FILTER_ROW =  bool(check_setting_int(CFG, 'GUI', 'display_filesize', 0))
```

Comment:
Yes it should... It's what I get for coding late at night and using copy
paste

On Wed, Apr 8, 2015 at 7:32 AM, Alexandre Beloin <notifications@github.com>
wrote:

> In sickbeard/__init__.py
> <https://github.com/SiCKRAGETV/SickRage/pull/1724#discussion_r27974860>:
>
> > @@ -1126,6 +1127,7 @@ def path_leaf(path):
> >          TIMEZONE_DISPLAY = check_setting_str(CFG, 'GUI', 'timezone_display', 'network')
> >          POSTER_SORTBY = check_setting_str(CFG, 'GUI', 'poster_sortby', 'name')
> >          POSTER_SORTDIR = check_setting_int(CFG, 'GUI', 'poster_sortdir', 1)
> > +        FILTER_ROW =  bool(check_setting_int(CFG, 'GUI', 'display_filesize', 0))
>
> @Thraxis <https://github.com/Thraxis> Shouldn't that be filter_row
> instead of display_filesize?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SiCKRAGETV/SickRage/pull/1724/files#r27974860>.
>